tests/components/prometheus/test_init.py

```diff
@@ -87,6 +88,77 @@
 PROMETHEUS_PATH = "homeassistant.components.prometheus"
 
 
+class EntityMetric:
```

Comment:
These helpers make the tests harder to read. Instead of just being able to read the tests I need to go look at this helper and try to understand how it works. It's not helpful in a project like Home Assistant with thousands of integrations if every integration designs its own test helpers and moves assertions to those.